converters/fromZigbee.js

```diff
@@ -519,6 +519,21 @@ const converters = {
                 }
             }
 
+            if (msg.data.hasOwnProperty('options')) {
+                const lightColorOptions = parseInt(msg.data['options']);
+                if (typeof lightColorOptions === 'number') {
```

Comment:
There is no need to do these checks and `parseInt()`, if the property is present it will always be a number. This can be simplified to:

```js
if (msg.data.hasOwnProperty('options')) {
  result.light_color_options = {execute_if_off: ((msg.data.options & 1<<0) > 0)};
}
```